File: Tasks.Infrastructure/Entities/Task.cs
using System;
using System.Collections.Generic;

namespace Tasks.Infrastructure.Entities;

public partial class Task
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public int UserId { get; set; }

    public int StatusId { get; set; }

    public DateOnly StartDate { get; set; }

    public DateOnly EndDate { get; set; }

    public int Duration { get; set; }

    public DateTime CreationDate { get; set; }

    public DateTime? UpdateDate { get; set; }

    public string? AdditionalInfo { get; set; }

    public virtual Status Status { get; set; } = null!;

    public virtual User User { get; set; } = null!;
}

File: TasksWebApp/src/app/classes/Task.ts
export interface Task extends TaskCreate{    
    creationDate:Date,        
    statusName:string,    
    userName:string,
    additionalInfo:string
}

export interface TaskCreate{
    id:number,
    name:string,    
    duration:number,
    statusId:number,    
    userId:number,    
}